lib/src/exception/exception.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {SourceSpan} from './span';
+
+export class SassException extends Error {
+  span?: SourceSpan;
+
+  constructor(options: {message: string; span?: SourceSpan; stack?: string}) {
```

Comment:
I'd make `message` a positional argument, both to match `new Error()` and because it's always required and unlikely to be ambiguous with the other arguments.